lib/compress/zstd_compress.c

```diff
@@ -20,6 +20,7 @@
 #include "fse.h"
 #define HUF_STATIC_LINKING_ONLY
 #include "huf.h"
+#include "zstd_compress_superblock.h"
```

Comment:
nit: very minor,

the general idea is that `#include "*.h"` should be listed from more generic to more specific,
in which case, I would expect `"zstd_compress_superblock.h"` to come last, or at least after  `"zstd_compress_internal.h"`.